common/persistence/nosql/nosqlplugin/cassandra/db.go

```diff
@@ -40,15 +40,38 @@ type cdb struct {
 
 var _ nosqlplugin.DB = (*cdb)(nil)
 
+type cassandraDBOption func(*cdb)
+
+func dbWithClient(client gocql.Client) cassandraDBOption {
```

Comment:
Seems like this function is only used in tests, I wonder is it good idea to put it in test file instead?

Reply:
it's used in the constructor below. making the contract more extensible for testing